<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaySoundBusinessActionWidget</class>
 <widget class="QWidget" name="PlaySoundBusinessActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="playToClient">
       <property name="text">
        <string>Play to user</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selectLayout" stretch="1,0">
     <item>
      <widget class="QComboBox" name="pathComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="manageButton">
       <property name="text">
        <string>Manage...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="testLayout">
     <item>
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Test...</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pathComboBox</tabstop>
  <tabstop>manageButton</tabstop>
  <tabstop>volumeSlider</tabstop>
  <tabstop>testButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
